services/worker-manager/src/providers/google.js

```diff
@@ -229,23 +229,31 @@ class GoogleProvider extends Provider {
       });
     }
 
-    const toSpawn = await this.estimator.simple({
+    let toSpawn = await this.estimator.simple({
       workerPoolId,
       ...workerPool.config,
-      capacityPerInstance: 1, // TODO (bug 1587234) Hardcoded for now until estimator updates
-      running: workerPool.providerData[this.providerId].running,
+      runningCapacity: workerPool.providerData[this.providerId].running,
     });
 
-    await Promise.all(new Array(toSpawn).fill(null).map(async i => {
+    if (toSpawn === 0) {
+      return; // Nothing to do
+    }
+
+    const cfgs = [];
+    while (toSpawn> 0) {
```

Comment:
whitespace around operator (this has been driving me nuts in contributors' code, too.. I'll try to get eslint to watch for this)